examples/kaniko.groovy

```diff
@@ -0,0 +1,43 @@
+/**
+ * This pipeline will build and deploy a Docker image with Kaniko
+ * https://github.com/GoogleContainerTools/kaniko
+ * without needing a Docker host
+ *
+ * You need to create a jenkins-docker-cfg secret with your docker config
+ * as described in
+ * https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-secret-in-the-cluster-that-holds-your-authorization-token
+ */
+
+ def label = "kaniko-${UUID.randomUUID().toString()}"
+
+ podTemplate(name: 'kaniko', label: label, yaml: """
+ kind: Pod
+ metadata:
+   name: kaniko
+ spec:
+   containers:
+   - name: kaniko
+     image: csanchez/kaniko:jenkins # we need a patched version of kaniko for now
```

Comment:
Is this because of GoogleContainerTools/kaniko#129 or something else?

Reply:
No, need cat and nohup in the image https://github.com/carlossg/kaniko/blob/jenkins/deploy/Dockerfile_jenkins